Examples/Examples/FloatingPanelExampleView.swift

```diff
@@ -39,26 +39,35 @@ struct FloatingPanelExampleView: View {
             viewpoint: initialViewpoint
         )
         .floatingPanel(selectedDetent: $selectedDetent, isPresented: isPresented) {
-            switch demoContent {
+            switch demoContent! {
```

Comment:
Instead of having `demoContent` be optional and force-unwrapping it here, can we just default it to something, maybe `.list`?

Reply:
Yes, I've reverted this change. It wasn't 100% necessary.